assets/scss/comments-and-meta.scss

```diff
@@ -7,35 +7,150 @@
   margin-bottom: 1rem;
   .comment-list {
     margin-bottom: 1rem;
+    background-color: #f0f4f5;
     .nhsuk-list-panel__item {
-      border-bottom: 1px solid #d8dde0;
       display: block;
-      padding: 12px 16px;
+      padding: 1.5rem;
+      background-color: #f0f4f5;
+      .children {
+        margin-left: 0;
+        margin-right: 0;
+        padding-left: 0;
+        .comment {
+          padding-right: 0;
+        }
+      }
       .comment-body {
+        display: flex;
+        .comment_body_comment {
+          display: block;
+          text-align: left;
+          border-radius: 3px;
+          order: 3;
+          line-height: 1.5;
+          padding: 18px 40px 18px 22px;
+          width: 70%;
+          position: relative;
+          word-wrap: break-word;
+          background-color: $color_nhsuk-white;
+          &:after {
+            content: '';
+            position: absolute;
+            width: 15px;
+            height: 15px;
+            border-radius: 50%;
+            background-color: #ffffff;
+            border: 2px solid #f3f3f3;
+            left: -8px;
+            top: 18px;
+          }
+        }
         .vcard {
-          float: right;
+          order: 1;
+          width: 17%;
           .fn {
             font-style: normal;
           }
+          cite {
+            display: block;
+            text-decoration: none;
+            color: #656c71;
+            font-weight: bold;
+            text-overflow: ellipsis;
```

Comment:
That is perfectly fine but perhaps not the best UI experience

Reply:
It will only show in exceptional circumstances where user name is incredibly long (as in exceeding 128 characters)